Library/Homebrew/startup/env_var.rb

```diff
@@ -0,0 +1,23 @@
+# typed: true
+# frozen_string_literal: true
+
+# @api private
+class MissingEnvironmentVariables < RuntimeError
+  attr_reader :env
+
+  def initialize(env)
+    super("#{env} was not exported! Please call bin/brew directly!")
+    @env = env
+  end
+end
+
+# Helper module for getting environment variables which must be set.
+#
+# @api private
+module EnvVar
+  def self.[](env)
+    raise MissingEnvironmentVariables, env unless ENV[env]
+
+    ENV.fetch(env)
+  end
+end
```

Comment:
I'd rather we just delegated this all to `ENV.fetch` instead and remove this class.

Reply:
Worth noting this already existed before but yeah I guess the answer to this depends on what we want as an error message.

Reply:
Yeh, kill it all I reckon. Was fine with class being around before but we've got a more resilient approach now (re-`exec`ing when necessary in both Bash and Ruby) that seems preferable than it being used more widely. I should have probably just used `ENV.fetch` the first time, honestly.